openapi/openapi-generator/Dockerfile

```diff
@@ -56,6 +40,21 @@ RUN chown root:root /etc/apt/sources.list.d/microsoft-prod.list
 RUN apt-get update
 RUN apt-get install -yy -q dotnet-hosting-2.0.8
 
+ARG OPENAPI_GENERATOR_COMMIT
```

Comment:
Does moving the ARGs affect the caching? I feel like it is clearer if they are at the top.

Reply:
According to the documentation, when the ARG layer is changed, all the following layers of the cache will invalidate.


### Reference: 
https://docs.docker.com/engine/reference/builder/#impact-on-build-caching
https://stackoverflow.com/questions/41593135/does-putting-arg-at-top-of-dockerfile-prevent-layer-re-use

Reply:
@brendanburns

Reply:
ok, thanks for the explanation.